src/routes/console/+layout.ts

```diff
@@ -16,7 +16,7 @@ export const load: LayoutLoad = async ({ fetch, depends }) => {
     const data = await response.json();
 
     return {
-        consoleVariables: sdk.forConsole.console.variables(),
+        consoleVariables: await sdk.forConsole.console.variables(),
```

Comment:
I'm not sure if it's worth it because:

1. it complicates the rest of the code
2. the variables are fetched 20ms after version even before this change

What could be useful is fetching both the version and the variables concurrently.